.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version
+      - image: circleci/golang:1.9
```

Comment:
Should this not be at least 1.11 (for new modules behavior)?

Reply:
good call!